lib/signaling/v2/peerconnection.js

```diff
@@ -463,6 +463,38 @@ class PeerConnectionV2 extends StateMachine {
     return true;
   }
 
+  /**
+   * Updates scaleResolutionDownBy for encoding layers.
+   * @param {number} width
+   * @param {object} encodings
+   */
+  _updateEncodings(width, encodings) {
+    // NOTE(mpatwardhan): All the simulcast encodings in Safari have
+    // the same resolution. So, we make sure that the lower layers have
+    // lower resolution, as seen in Chrome.
+    // let maxScaleDown = 4;
+    // +=========+========================+
+    // |  width  | scaleResolutionDownBy  |
+    // +=========+========================+
+    // | 0-359   |  1, inactive, inactive |
+    // +---------+------------------------+
+    // | 360-719 |  2, 1, inactive        |
+    // +---------+------------------------+
+    // | 720+    |  4, 2, 1               |
+    // +---------+------------------------+
+
+    const activeLayers = Math.min(encodings.length, width <= 359 ? 1 : width <= 719 ? 2 : 3);
+    encodings.forEach((encoding, i) => {
+      if (i >= activeLayers) {
+        encoding.active = false;
+        this._log.debug(`Layer ${i} active = false`);
+      } else {
+        encoding.scaleResolutionDownBy = 1 << (activeLayers - i - 1);
+        this._log.debug(`Layer ${i} scaleResolutionDownBy = ${encoding.scaleResolutionDownBy}`);
+      }
```

Comment:
I don't think we need log statements here. Plus, I think this can be rewritten as:
```suggestion
      encoding.active = i < activeLayers;
      if (encoding.active) {
        encoding.scaleResolutionDownBy = 1 << (activeLayers - i - 1);
      }
```

Reply:
I have taken your suggestions. But kept the log, as I do think its useful info - if customers report any issues with logs..